docs/_static/custom_cookietemple.css

```diff
@@ -13,24 +13,6 @@ div, span, code {
     background-color: #005fff !important;
 }
 
```

Comment:
Why is all this css code removed?

Reply:
I don't know. For some reason the cookietemple css and this one were out of sync.

Which one is the latest one?

Reply:
I added the more complete one to CT AND to the common files